libraries/standard/mqtt/integration-test/mqtt_system_test.c

```diff
@@ -248,26 +267,41 @@ static void eventCallback( MQTTContext_t * pContext,
 
 /*-----------------------------------------------------------*/
 
+uint32_t getTimeMs( void )
```

Comment:
This doesn't really belong in this file, as platform specific code should be in `platform/posix`